doc/source/guide/style.rst

```diff
@@ -45,18 +46,19 @@ We now see that the displayed visualizations adopt these new imported settings.
   :align: center 
 
 Similarly, we can change the plot configurations for Matplotlib charts as well.
-The plot_config attribute for Matplotlib charts takes in both the figure and axis as parameters.
+The plotting_style attribute for Matplotlib charts takes in both the figure and axis as parameters.
```

Comment:
Why take in both figure and axis? instead of just one of these? The example below should motivate this if they are both necessary.

Reply:
figure can set attributes like figsize, while axis doesn't have the capabilities. axes can set attributes like plot titles